benchmarks/test_benchmark_coo.py

```diff
@@ -3,16 +3,16 @@
 import numpy as np
 
 
-class MatrixMultiplySuite:
-    def setup(self):
-        rng = np.random.default_rng(0)
-        self.x = sparse.random((100, 100), density=0.01, random_state=rng)
-        self.y = sparse.random((100, 100), density=0.01, random_state=rng)
+def test_MatrixMultiplySuite(benchmark):
+    rng = np.random.default_rng(seed=42)
+    x = sparse.random((3, 3), density=0.01, random_state=rng)
+    y = sparse.random((3, 3), density=0.01, random_state=rng)
```

Comment:
3 is way too small: We'd like to have something statiscally significant. This is so small most of the measurements will be noise or overhead of the Python call, rather than actual number crunching.

Reply:
What should it be? I took 3 from `finch_example.py`